src/env.ts

```diff
@@ -40,7 +40,11 @@ export default envalid.cleanEnv(
 		}),
 		DATABASE_LOGGING: envalid.bool({
 			default: false
-		})
+		}),
+
+		GRAPHQL_URI: envalid.url({
```

Comment:
I think GRAPHQL_URI is not url. It can be IP address or host. Will it work?

Reply:
localhost:5000 - it is correct url, but it is not correct host.